server/routes/overview/viewArchivedGoalsController.test.ts

```diff
@@ -35,14 +35,11 @@ describe('viewArchivedGoalsController', () => {
   })
   it('should get view archived goals page', async () => {
     // Given
-    const expectedTab = 'view-archived-goals'
```

Comment:
The PR in general LGTM 👍  (and I will approve it 😁 ), but the removal of the expected tab from here and the view class itself has made me stop and think....

The `tab` property in the view class (and therefore the expected tab in the tests) is there so that we have a view class for each "tab" on the Overview page.
I'd assumed that the new Archived Goals page was a tab on the Overview page, hence the need for the property etc.
So seeing you remove it here made me look at the designs.
You are right, its not a tab on the Overview page, so you are right to remove the property 👍 

But .... if it's not a tab on the Overview page, and is a page in it's own right, then I think this controller and view class etc are in the wrong folder in the source tree (in that it's nothing to do with the Overview page)

Let's go with this PR as it is 👍 , and then do another one to refactor `viewArchivedGoalsController` and associated classes etc into a better folder structure, outside of the `overview` structure 👍

Reply:
ok - I left it here on the basis that we're shortly changing the overview and goals/archived/completed will be sub-tabs of the overview. No biggie either way to me.

Reply:
Ah, in that case, yes. If they will soon become subtabs of the Overview page, then yes, its all in the right place 😁 
Let's leave it for now 😁 👍